packages/logging_cloudwatch/aws_logging_cloudwatch/lib/src/cloudwatch_logger_plugin.dart

```diff
@@ -5,6 +5,8 @@ import 'dart:async';
 import 'dart:math';
 
 import 'package:amplify_core/amplify_core.dart';
+// ignore: invalid_use_of_internal_member, implementation_imports
```

Comment:
Looking through our repo I don't see any precedent for ignoring these warnings.  What is the reason we need to ignore this warning here?  "invalid_use_of_internal_member"